README.md

```diff
@@ -31,4 +34,38 @@ Url: http://localhost:5173 # or live version https://defeder-remix-deploy.netlif
 Type of connection: Iframe
 Location in Remix: Side Panel
 ```
-5. You should see the plugin added to the sidebar (new icon with ? symbol).
\ No newline at end of file
+5. You should see the plugin added to the sidebar (new icon with ? symbol).
+
+## Testing in Contracts Wizard
+
+For testing in Contracts Wizard, you must also run the Contracts Wizard UI locally to point to your local plugin.
+
+1. Run Contracts Wizard locally.
+ - a. Go to (https://github.com/OpenZeppelin/contracts-wizard)[Contracts Wizard Repo].
```

Comment:
Fixes link formatting (should also apply a similar fix for Remix link on line 26)
```suggestion
 - a. Go to [https://github.com/OpenZeppelin/contracts-wizard](Contracts Wizard Repo).
```